app/models/comment.rb

```diff
@@ -1,4 +1,10 @@
 class Comment < ApplicationRecord
-  belongs_to :author
+  belongs_to :author, class_name: 'User'
   belongs_to :post
+
+  after_create :increment_comments_counter
+
+  def increment_comments_counter
+    post.increment!(:comments_counter)
```

Comment:
- [ ] [OPTIONAL] Nice work using the rails callback method. However, you will need the `after_save` method instead of the `after_create` method here. It is also advisable to make the `increment_comments_counter` method private. when using the rails active callbacks `after_save` method to call it. :+1:

Reply:
done